hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Nice, that makes sense.  Will look at a few more scripts as see if I can better line up with any other conventions.